sphinx/ext/imgmath.py

```diff
@@ -126,15 +130,22 @@ def compile_math(latex: str, builder: Builder) -> str:
     # build latex command; old versions of latex don't have the
     # --output-directory option, so we have to manually chdir to the
     # temp dir to run it.
-    command = [builder.config.imgmath_latex, '--interaction=nonstopmode']
+    command = [builder.config.imgmath_latex]
+    # assume that executable is on the PATH (i.e., don't handle /path/to/tectonic)
+    if builder.config.imgmath_latex not in ['tectonic']:
```

Comment:
maybe use `path.basename(build.config.imgmath_latex)`.  As this is again used below maybe we need a variable to store the user command. The `imgmath_latex` is [documented](https://www.sphinx-doc.org/en/master/usage/extensions/math.html#confval-imgmath_latex) to have to be either a plain command name (without options) or a full path.

Reply:
Done. Personally I would have preferred to use 
```python
from pathlib import Path
imgmath_latex_name = Path(builder.config.imgmath_latex).name
# ...
```
but I prefer to be consistent with the rest of the code.